.github/pull_request_template.md

```diff
@@ -1,29 +1,22 @@
-<!--- Thanks for contributing to SSW.Rules! -->
-## Reason for change (Issue, Email, conversation + reason, etc)
-<!-- 
-- Example 1 - Relates to #{{ ISSUE NUMBER }}
-- Example 2 - As per my conversation with...
-- Example 3 - From email, subject...
-- Example 4 - I noticed that... 
--->
-{{ REASON FOR CHANGE - WHAT WAS THE INITIATED FOR THIS CHANGE }}
+**Reason for change (PBI link, Email Subject, conversation + reason, etc)**
 
-{{ SUMMARY OF CHANGES }}
+> 1. What triggered this change?
```

Comment:
```suggestion
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)
```

Reply:
@bradystroud as per our conversation, If combine these lines this could look even cleaner.

Let's merge this and see if it helps 🙂